bundles/org.openhab.binding.easee/README.md

```diff
@@ -0,0 +1,170 @@
+| passord                 | yes      | Your password to login at Easee Cloud service.                                                                                                                                                                                                                                                                                                           |
+| siteId                  | yes      | The ID of the site containing the wallbox(es) and circuit(s) that should be integrated into openHAB. The ID of your site can be found out via the sites overview (https://easee.cloud/sites). You just need to click on one of the sites listed there, the id will be part of the URL which is then opened. It will be a number with typically 6 digits. |
+| dataPollingInterval     | no       | interval (seconds) in which live data values are retrieved from the Easee Cloud API. (default = 120)                                                                                                                                                                                                                                                     |
+
+## Thing configuration
+
+It is recommended to use auto discovery which does not require further configuration.
+If manual configuration is preferred you need to specify configuration as below.
+
+### Charger
+
+| Configuration Parameter | Required | Description                                                                                                            |
+|-------------------------|----------|------------------------------------------------------------------------------------------------------------------------|
+| id                      | yes      | The id of the charger that will be represented by this thing.                                                          |
+
+### Mastercharger
+
+| Configuration Parameter | Required | Description                                                                                                            |
+|-------------------------|----------|------------------------------------------------------------------------------------------------------------------------|
+| id                      | yes      | The id of the charger that will be represented by this thing.                                                          |
+| circuitId               | yes      | The id of the circuit that is controlled by this charger.                                                              |
+
+## Channels
+
+The binding only supports a subset of the available endpoints that is provided by the Easee Cloud API.
+The tables below shows all channels that are available and which of them are writable.
+The setting that start with "dynamic" can be changed frequently, the other are written to flash and thus should not be changed too often as this could result in damage of your flash.
```

Comment:
```suggestion
The settings that start with "dynamic" can be changed frequently, the others are written to flash and thus should not be changed too often as this could result in damage of your flash.
```

Reply:
fixed